organisations/organisations.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+
+package organisations;
+
+option go_package = "grpc/organisationspb";
+
+import "google/protobuf/empty.proto";
+
+message Organisation {
+    string id = 1;
+
+    string name = 2;
+
+    Member creator = 3;
+    
+    repeated Member members = 4;
+}
+
+message Member {
+    string id = 1;
```

Comment:
I would rename this `id` with `account_id` because it's a foreign key. Correct me if I'm wrong.